src/streaming/controllers/MediaController.js

```diff
@@ -72,6 +85,43 @@ MediaPlayer.dependencies.MediaController = function () {
              };
          },
 
+         selectInitialTrack = function(tracks) {
+             var initTrack,
+                 max = 0,
+                 tmp,
+                 idx = 0,
+                 mode = this.getSelectionModeForInitialTrack();
+
```

Comment:
@JibberJim, thanks for the suggestion. I have updated selection algorithm:
 https://github.com/Dash-Industry-Forum/dash.js/commit/c2c6910d103665283c812373e87953c11f774e77
Now, if there are several tracks with the same highest bitrate (widest range) the player checks which one has widest range (highest bitrate).